### PR TITLE
Update 1.2.1

### PR DESCRIPTION
**Pull request by Jrice1317**
- Removed noarch and updated python pinnings
- Added skip
- Changed flit to flit-core (i.e. https://github.com/Kozea/tinycss2/blob/master/pyproject.toml#L2)
- Added wheel to host section and pip requires and pip check to test section (linter)
- Updated home and doc url
- Added license and doc source urls

Jira: https://anaconda.atlassian.net/browse/PKG-668
https://github.com/Kozea/tinycss2/blob/v1.2.1/pyproject.toml

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
+  jaidarice: nbconvert-update
```
Comment (katietz) [inline abs.yaml]: please don't miss to remove this channel specification on final build

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,44 +1,52 @@
...
 source:
-  url: https://pypi.io/packages/source/t/tinycss2/tinycss2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 8cff3a8f066c2ec677c06dbc7b45619804a6938478d9d73c284b29d14ecb0627
 
 build:
   number: 0
-  noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  #[py<37]
```
Comment (katietz) [inline recipe/meta.yaml]: please add a space after the '#' sign

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,44 +1,52 @@
...
 
 requirements:
   host:
-    - flit
-    - python >=3.5
+    - flit-core >=3.2,<4
```
Comment (katietz) [inline recipe/meta.yaml]: please don't use >= in host section.  It should be 'flit-core 3.2' instead.  Version ranges are only for the 'run' section to be used

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,44 +1,52 @@
...
+    - python
     - pip
     - pytest-runner
     - webencodings >=0.4
```
Comment (katietz) [inline recipe/meta.yaml]: please fix this too.  see comment above

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,44 +1,52 @@
...
+    - wheel
```
Comment (katietz) [inline recipe/meta.yaml:25]: missing setuptools dependency.  Please add

File: recipe/meta.yaml [hunk 6]
```diff
@@ -1,44 +1,52 @@
...
   run:
-    - python >=3.5
+    - python
```
Comment (katietz) [inline recipe/meta.yaml:27]: I assume that you want to add flit-core here? or is that not required?  what is setup.py in sources tell you?

File: recipe/meta.yaml [hunk 7]
```diff
@@ -1,44 +1,52 @@
...
   license_file: LICENSE
+  license_url: https://github.com/Kozea/tinycss2/blob/v{{ version }}/LICENSE
   summary: Low-level CSS parser for Python
   description: |
     tinycss2 is a rewrite of tinycss with a simpler API, based on the more recent
     CSS Syntax Level 3 Specification.
-  doc_url: http://tinycss2.readthedocs.io
+  doc_url: https://doc.courtbouillon.org/tinycss2/stable/
   dev_url: https://github.com/Kozea/tinycss2
+  doc_source_url: https://github.com/Kozea/tinycss2/tree/v{{ version }}/docs
```
Comment (katietz) [inline recipe/meta.yaml]: agreed that linter gives here a warning, but having doc_source_url, and doc_url is simply too much ... the same thing about the license_url ... we provide a LICENSE in license_file already, why specifying it again?

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 5 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:25]: a build backend is flit-core, see https://github.com/Kozea/tinycss2/blob/853bc5a5661a59c7a9795a4e2758d0081808ce06/pyproject.toml#L3

File: recipe/meta.yaml (hunk 6 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:27]: `setup.py` doesn't exist, only pyproject.toml for flit-core

File: recipe/meta.yaml (hunk 4 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Can we try to build without `webencodings` in `host`? I guess it's only runtime dependency

File: recipe/meta.yaml (hunk 3 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: Ahhh, okay, thanks! I thought we only took the pinnings off of python, but that makes sense and I will remember that for future reference. Thank you!

File: recipe/meta.yaml (hunk 7 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: I agree, but I only provided it because linter was failing and those were the error messages I got.

**Review by ryanskeith — APPROVED** [review]
After incorporating the changes mention, it looks good to me.

**Review by varlackc — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]

File: abs.yaml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+channels:
```
Comment (katietz) [inline abs.yaml]: please get rid of this channel in abs

File: recipe/meta.yaml [hunk 9]
```diff
@@ -1,44 +1,53 @@
...
 requirements:
   host:
-    - flit
-    - python >=3.5
+    - flit-core 3.2
+    - python
     - pip
     - pytest-runner
+    - setuptools
     - webencodings >=0.4
```
Comment (katietz) [inline recipe/meta.yaml]: is this range for webencodings really necessary?  we should prefer to use no version ranges in host section, if possible

File: recipe/meta.yaml [hunk 10]
```diff
@@ -1,44 +1,53 @@
...
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/Kozea/tinycss2
+  home: https://www.courtbouillon.org/tinycss2
   license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE
+  license_url: https://github.com/Kozea/tinycss2/blob/v{{ version }}/LICENSE
```
Comment (katietz) [inline recipe/meta.yaml]: please remove this superflous license_url

File: recipe/meta.yaml [hunk 11]
```diff
@@ -1,44 +1,53 @@
...
   summary: Low-level CSS parser for Python
   description: |
     tinycss2 is a rewrite of tinycss with a simpler API, based on the more recent
     CSS Syntax Level 3 Specification.
-  doc_url: http://tinycss2.readthedocs.io
+  doc_url: https://doc.courtbouillon.org/tinycss2/stable/
   dev_url: https://github.com/Kozea/tinycss2
+  doc_source_url: https://github.com/Kozea/tinycss2/tree/v{{ version }}/docs
```
Comment (katietz) [inline recipe/meta.yaml]: please remove this superflous doc_source_url tag.  there is already a doc_url

**Review by katietz — CHANGES_REQUESTED** [review]

File: abs.yaml (hunk 8 above)
Comment (katietz) [reply abs.yaml]: still in abs.yaml?

File: recipe/meta.yaml (hunk 9 above)
Comment (katietz) [reply recipe/meta.yaml]: there is no 0.4 version available for webenconding.  Only version we have is actually 0.5.1

File: abs.yaml (hunk 8 above)
Comment (katietz) [reply abs.yaml]: I did it for you

File: recipe/meta.yaml (hunk 10 above)
Comment (katietz) [reply recipe/meta.yaml]: I did it for you

File: recipe/meta.yaml (hunk 11 above)
Comment (katietz) [reply recipe/meta.yaml]: I did it also for you

**Review by katietz — APPROVED** [review]
Now it is fine for me too

**Pull request merged**